cdk/lib/cdk-stack.ts

```diff
@@ -1,10 +1,24 @@
-import * as cdk from '@aws-cdk/core';
+import * as cdk from '@aws-cdk/core'
+import * as lambda from '@aws-cdk/aws-lambda'
+import {SqsEventSource} from '@aws-cdk/aws-lambda-event-sources'
+import * as sqs from '@aws-cdk/aws-sqs'
 
 export class CdkStack extends cdk.Stack {
   constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
-    super(scope, id, props);
-
-    // The code that defines your stack goes here
+    super(scope, id, props)
 
+    // Can add timeout, deadLetterQueue, memorySize etc. later to fine tune the created lambda
+    const queueSongLambda = new lambda.Function( this, 'QueueSongLambda', {
+      code: new lambda.AssetCode(`../queue-song-lambda/src`),
+      handler: 'Foo.foo',
+      runtime: lambda.Runtime.JAVA_8,
+    })
+    const songRequestsQueue = new sqs.Queue(this, 'SongRequestQueue', {
+      visibilityTimeout: cdk.Duration.seconds(30), //default
+      receiveMessageWaitTime: cdk.Duration.seconds(20)
```

Comment:
This is a message delay before the lambda receives it? Not sure if we need this 🤔

Reply:
Oh I didn't put the comment but I believe both of these should be default values. We can reduce the `receiveMessageWatiTime` to whatever we determine is best.

Reply:
If it's default, then whatever. Just figured I'd ask.